Mac OS X Build

It's time to start trying to build the `omnicore-0.0.10` branch on Mac OS X.

I just made a quick try using the instructions in `README.md`, that is:

    ./autogen.sh
    ./configure
    make

And the make is failing with the following error:

    Making all in src
      CXX      omnicore/libbitcoin_server_a-version.o
    omnicore/version.cpp:9:14: fatal error: 'build.h' file not found
    #    include "build.h"
                 ^
    1 error generated.
    make[2]: *** [omnicore/libbitcoin_server_a-version.o] Error 1
    make[1]: *** [all-recursive] Error 1  
    make: *** [all-recursive] Error 1

I'm using *OS X Yosemite* 10.10.3, *Xcode* 6.3.1, and have used *Homebrew* to install the prerequisites as described in [build-osx.md](https://github.com/bitcoin/bitcoin/blob/master/doc/build-osx.md)

--- a/src/Makefile.omnicore.include
+++ b/src/Makefile.omnicore.include
@@ -43,4 +43,4 @@ libbitcoin_server_a_SOURCES += \
   $(OMNICORE_CPP) \
   $(OMNICORE_H)
 
-libbitcoin_server_a-omnicore/version.$(OBJEXT): obj/build.h # build info
+omnicore/libbitcoin_server_a-version.$(OBJEXT): obj/build.h # build info

